### DISCUSSION TITLE
data show error

### DISCUSSION THREAD

**Discussion by AnkurDalwadi3112** [Bottelet/Flarepoint-crm#264] [General]
![error](https://user-images.githubusercontent.com/69509124/123509857-900a5700-d695-11eb-9d9f-65475c5476eb.png)
don't show table data in clients , projects, task , user page list

**Comment by Bottelet** [comment]
Hi @AnkurDalwadi3112  Please open this as an issue. Also, I can't reproduce it locally myself. Can you show me the errors n the console, and networks tab?
